lib/src/main/java/graphql/nadel/validation/NadelFieldValidation.kt

```diff
@@ -96,24 +98,22 @@ internal class NadelFieldValidation(
                 if (underlyingArg == null) {
                     MissingArgumentOnUnderlying(parent, overallField, underlyingField, overallArg)
                 } else {
-                    val unwrappedTypeIssues = typeValidation
-                        .validate(
```

Comment:
So turns out this call and similar were useless, no point calling back to `NadelTypeValidation.validate` as the same elements will always be validated anyway.

We just need to check the name is the same & assume the validation process will do its job.

Proof of this is in bf156ad370c4e084341ea68082152d602589cc22

There are some failures on the above commit, but these are expected failures

Our own tests

<img width="444" alt="image" src="https://github.com/user-attachments/assets/14c0ad79-9d10-4005-baca-1cd29335ac98">

For Central

<img width="1090" alt="image" src="https://github.com/user-attachments/assets/74b36d5d-058c-4ae1-8892-088666a1e89f">